game/theater/controlpoint.py

```diff
@@ -309,6 +310,7 @@ def __init__(
         self.captured_invert = False
         # TODO: Should be Airbase specific.
         self.has_frontline = has_frontline
+        self.delayed_start: timedelta = timedelta(seconds=0)
```

Comment:
It looks like you're creating a `QTimeEdit` per front but only a delay per base. The most recently set time will clobber the others for a multi-front base.